{"name": "整体增删改查用户", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'", "trace": "self = <WechatAPI.testcase.test_work.TestWework object at 0x00000208666FE198>\nuserid = 'userid3', name = '名字3', mobile = '13800000003'\n\n    @allure.story(\"通讯录管理\")\n    @allure.title(\"整体增删改查用户\")\n    @pytest.mark.parametrize(\"userid,name,mobile\",test_data())\n    def test_wechat(self,userid,name,mobile):\n    \n        # userid=\"zhangsan90\"\n        # name=\"张三思0\"\n        # mobile = \"13290900011\"\n        try:\n>           assert \"created\"==Wework().test_creat_member(userid,name,mobile)[\"errmsg\"]\n\nWechatAPI\\testcase\\test_work.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <WechatAPI.api.we_work.Wework object at 0x00000208666FE0F0>\n\n    def __init__(self):\n        self.token = Util().token() #不要忘了Util的()\n        self.params[\"token\"]=self.token #要记得替换token的变量\n    \n>       with open(\"../api/wework.yaml\",encoding=\"utf-8\")as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'\n\nWechatAPI\\api\\we_work.py:27: FileNotFoundError"}, "parameters": [{"name": "userid", "value": "'userid3'"}, {"name": "name", "value": "'名字3'"}, {"name": "mobile", "value": "'13800000003'"}], "start": 1647487023599, "stop": 1647487026715, "uuid": "9ffdf5e4-4685-4774-9bde-44aa5facca2a", "historyId": "b5c85de6003df0312ab78a391480e912", "testCaseId": "6fbae5519000b9212e1162caf7211d2e", "fullName": "WechatAPI.testcase.test_work.TestWework#test_wechat", "labels": [{"name": "story", "value": "通讯录管理"}, {"name": "parentSuite", "value": "WechatAPI.testcase"}, {"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ZXJ"}, {"name": "thread", "value": "61948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "WechatAPI.testcase.test_work"}]}